rotkehlchen/data_import/importers/nexo.py

```diff
@@ -128,6 +133,12 @@ def _consume_nexo(
                 log.debug(f'Ignoring nexo entry {csv_row} with negative interest')
                 return
 
+            # from the csv a user shared, some entries with type of
+            # interest and referral bonus did not contain an output amount
+            # so in such cases, fall back to the input amount.
+            if entry_type in {'Interest', 'Referral Bonus'} and amount == ZERO:
```

Comment:
Apart from the comment what else makes no sense is this entire elif code.

The comment says in some cases the OutputAmount is empty. And goes to mention again this `'Interest'` and `'Referral Bonus'` and in the case of which you should take and force positive the `Input Amount`.

But ... the comment said in these cases OutputAmount is missing. You later said, and the code you wrote assumes it's not missing but it's empty. In both cases `deserialize_asset_amount(csv_row['Output Amount'])` will raise a ValueError and we will never get here.

So this seems completely broken.

Did you by any chance mean here that in those cases the output amount is zero (and not missing, '', or "did not contain an output amount) ?